public/Payment.html

```diff
@@ -36,12 +39,45 @@
             }
             
             fetch(payUrl).then(res => {
-                window.history.back();
+                console.log(res);
```

Comment:
remove commented-out code as well as console.log

Reply:
There is code that needs to be added here, that function is still under construction